examples/tokens/token_dissociate_transaction.py

```
@@ -47,7 +47,7 @@ def create_new_account(client, operator_id, operator_key):
         print(f"✅ Success! Created new account with ID: {recipient_id}")
         return client, operator_key, recipient_id, recipient_key, operator_id
 
-    except (ValueError, RuntimeError) as e:
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)